src/image/Image.cpp

```diff
@@ -4,15 +4,24 @@
 #include <pfm/pfm_input_file.hpp>
 #include <pic/pic_input_file.hpp>
 
+#include <QImage>
+
 namespace hdrv {
 
-Image::Image(int w, int h, int c, std::unique_ptr<float[]> data)
+Image::Image(int w, int h, int c, Format f, const std::vector<uint8_t>&& data)
```

Comment:
const rvalue ref doesn't make any sense. Just passing by value is fine here unless you're _really_ keen on preventing accidental copy.
Any particular reason for changing to vector instead of unique_ptr<uin8_t[]>?